[System] Fix metrics overview dashboard

milan-elastic opened the pull request:
- Bug
<!-- Type of change
Please label this PR with one of the following labels, depending on the scope of your change:

- Enhancement
- Breaking change
- Deprecation
-->

## Proposed commit message
- Fixed the Table chart visualization, where CPU usage is comming as '-'.
- Changed palette param from 'Percent' to 'Number' for Table chart visualization.
- Added data_stream.dataset filter at panel for `Top hosts by CPU usage over time` and `Top hosts by memory usage over time`.

<!-- Mandatory
Explain here the changes you made on the PR.

Please explain:

- WHAT: patterns used, algorithms implemented, design architecture, message processing, etc.
- WHY:  the rationale/motivation for the changes

This text will be pasted into the squash dialog when the change is committed and will be
a long term historical record of the change to help future contributors understand the
change, please help them by making it clear and comprehensive, they may be you.

If the commit title is adequate to describe both of these things, The text here may be omitted
or replaced with "See title". The title of the PR will be used as the commit message title when
the merge is made and the "See title" marker will be removed if present.

The text here and the PR title will be subject to the PR review process.
-->

## Related Issues
- closes #9733 

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Screenshots
Before

![screencapture-my-deployment-4520e7-kb-us-central1-gcp-cloud-es-io-9243-app-dashboards-2024-05-03-14_42_29](https://github.com/elastic/integrations/assets/118723373/a43fb27c-2899-45e4-a3cc-6e9257e620af)

After 

![screencapture-127-0-0-1-64942-app-dashboards-2024-05-03-13_28_29](https://github.com/elastic/integrations/assets/118723373/e93905b8-afb6-4b67-b37f-d7da942b3116)

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/11023)
* Commit: b782ac5120b4afc594dc746ce73166b36e407140

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11023","number":11023,"commit":"b782ac5120b4afc594dc746ce73166b36e407140"}],"number":11023}
buildkite-pr-comment-->

milan-elastic commented:
While comparing the before migration system metrics dashboards to this [PR](https://github.com/elastic/integrations/pull/6743/files), I have observed that the fields are getting changed. And from this change, the metrics visualization and the table chart for CPU usage are showing different values! Were these changes expected? 
cc: @drewdaemon

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/11044)
* Commit: 41510b0671eb1dc2fdc49c9cbd7e8e3be8a1dff4

### History
* :green_heart: [Build #11023](https://buildkite.com/elastic/integrations/builds/11023) succeeded b782ac5120b4afc594dc746ce73166b36e407140

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11044","number":11044,"commit":"41510b0671eb1dc2fdc49c9cbd7e8e3be8a1dff4"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11023","number":11023,"commit":"b782ac5120b4afc594dc746ce73166b36e407140"}],"number":11044}
buildkite-pr-comment-->

ishleenk17 commented:
> While comparing the before migration system metrics dashboards to this [PR](https://github.com/elastic/integrations/pull/6743/files), I have observed that the fields are getting changed. And from this change, the metrics visualization and the table chart for CPU usage are showing different values!
> 
> Before Migration ![image (7)](https://private-user-images.githubusercontent.com/118723373/327342683-5798b976-d517-40b5-bf23-9445cf26e202.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3MTc3MTksIm5iZiI6MTcxNDcxNzQxOSwicGF0aCI6Ii8xMTg3MjMzNzMvMzI3MzQyNjgzLTU3OThiOTc2LWQ1MTctNDBiNS1iZjIzLTk0NDVjZjI2ZTIwMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwM1QwNjIzMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZmY1MmEzMDU3YjY3ZTQwZTUzY2RlM2E2Mzc2MjczODgzZDk1MjgwOThkYWQ4NjM0MzI2NGQ1YTQzZDI4YmI2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ynJMW-eRyzyulYnC_caSVYyDlKapwXo-xR_B0CIt_Sk)
> 
> After Migration ![image (8)](https://private-user-images.githubusercontent.com/118723373/327342737-970b8d7f-15b9-483f-a7b2-c116ef05416f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3MTc3MTksIm5iZiI6MTcxNDcxNzQxOSwicGF0aCI6Ii8xMTg3MjMzNzMvMzI3MzQyNzM3LTk3MGI4ZDdmLTE1YjktNDgzZi1hN2IyLWMxMTZlZjA1NDE2Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwM1QwNjIzMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wOTgxYThkZjdiZmJkMGI3MzU2MDVhOWQzOWZkNGQ1YWIyYWY4MjczNTkwZjUwNmJmMjM2YWJlNTYxNTk5NjJkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.96sSYXVisY_nq3yrzzyEPr8JzVj_wqS81I7P6QQzVDA)
> 
> Were these changes expected? cc: @drewdaemon

Do we have closure on this ?
Do we know the reason of CPU values being shown as 0% ?

ishleenk17 commented:
@milan-elastic : Was this issue reprodicible where in 10s we don't see the CPU values?

milan-elastic commented:
> > While comparing the before migration system metrics dashboards to this [PR](https://github.com/elastic/integrations/pull/6743/files), I have observed that the fields are getting changed. And from this change, the metrics visualization and the table chart for CPU usage are showing different values!
> > Before Migration ![image (7)](https://private-user-images.githubusercontent.com/118723373/327342683-5798b976-d517-40b5-bf23-9445cf26e202.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3MTc3MTksIm5iZiI6MTcxNDcxNzQxOSwicGF0aCI6Ii8xMTg3MjMzNzMvMzI3MzQyNjgzLTU3OThiOTc2LWQ1MTctNDBiNS1iZjIzLTk0NDVjZjI2ZTIwMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwM1QwNjIzMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZmY1MmEzMDU3YjY3ZTQwZTUzY2RlM2E2Mzc2MjczODgzZDk1MjgwOThkYWQ4NjM0MzI2NGQ1YTQzZDI4YmI2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ynJMW-eRyzyulYnC_caSVYyDlKapwXo-xR_B0CIt_Sk)
> > After Migration ![image (8)](https://private-user-images.githubusercontent.com/118723373/327342737-970b8d7f-15b9-483f-a7b2-c116ef05416f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3MTc3MTksIm5iZiI6MTcxNDcxNzQxOSwicGF0aCI6Ii8xMTg3MjMzNzMvMzI3MzQyNzM3LTk3MGI4ZDdmLTE1YjktNDgzZi1hN2IyLWMxMTZlZjA1NDE2Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwM1QwNjIzMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wOTgxYThkZjdiZmJkMGI3MzU2MDVhOWQzOWZkNGQ1YWIyYWY4MjczNTkwZjUwNmJmMjM2YWJlNTYxNTk5NjJkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.96sSYXVisY_nq3yrzzyEPr8JzVj_wqS81I7P6QQzVDA)
> > Were these changes expected? cc: @drewdaemon
> 
> Do we have closure on this ? 

No, not yet! I am looking forward to get a confirmation from @drewdaemon on this.

> Do we know the reason of CPU values being shown as 0% ?

The panel previously displayed "-" instead of the actual value, resulting in 0% being shown. This issue has been resolved as part of this pull request.

milan-elastic commented:
> @milan-elastic : Was this issue reprodicible where in 10s we don't see the CPU values?

Yes, it is reproducible and fixed now as part of this PR

cmacknz commented:
The total system CPU usage definitely seems more correct.

@fearful-symmetry can you sanity check the correct CPU metrics are being used here?

drewdaemon commented:
A side-note I wanted to state out-loud here. Completely removing the reduced time range setting will mean that the table may show out-of-date metrics and this may not be clear to the user. For example, maybe the last CPU metric for host `A` was reported at 80% yesterday. My dashboard time range is set to the last 7 days. I will see 80% as the "current" value even though it was reported yesterday.

Not saying it's the wrong choice to remove this, just saying there's a trade-off. Another possibility would be increasing the reduced time range to a larger window to account for slight variances in the ingest frequency.

cmacknz commented:
> Not saying it's the wrong choice to remove this, just saying there's a trade-off. Another possibility would be increasing the reduced time range to a larger window to account for slight variances in the ingest frequency.

Increasing the reduced time range seems like it might be a better choice. Showing 1 week old data for metrics feels wrong. 

The data collection interval defaults to 10s, so at most you'd have one sample in that period. Using a longer one makes more sense. What value to use feels a bit arbitrary, maybe 15m instead of 10s to guarantee we have something unless there is a real problem?

I think in an ideal world we'd want this to be some fixed multiple of the configured collection period, but I'm not sure that's possible.

drewdaemon commented:
> What value to use feels a bit arbitrary, maybe 15m instead of 10s to guarantee we have something unless there is a real problem?

FWIW, I think this makes sense.

> I think in an ideal world we'd want this to be some fixed multiple of the configured collection period, but I'm not sure that's possible.

Not possible today. But, I think variables in the integration assets would be powerful.

milan-elastic commented:
After our team discussion, we've decided to adjust the time range to 15 minutes. While this won't always ensure complete data population,But any day it's a better option than sticking with a reduced time range of 10 seconds.
cc: @tommyers-elastic @ishleenk17 @lalit-satapathy

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/11275)
* Commit: 48337698515e1cbf889045c7ee352a5ddceb7101

### Failed CI Steps
* [:pipeline::arrow_up: Upload Pipeline: .buildkite/pipeline.yml](https://buildkite.com/elastic/integrations/builds/11275#018f58ab-428b-4776-9ea6-653b9cff5f5d)

### History
* :green_heart: [Build #11044](https://buildkite.com/elastic/integrations/builds/11044) succeeded 41510b0671eb1dc2fdc49c9cbd7e8e3be8a1dff4
* :green_heart: [Build #11023](https://buildkite.com/elastic/integrations/builds/11023) succeeded b782ac5120b4afc594dc746ce73166b36e407140

cc @milan-elastic

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11275","number":11275,"commit":"48337698515e1cbf889045c7ee352a5ddceb7101"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11044","number":11044,"commit":"41510b0671eb1dc2fdc49c9cbd7e8e3be8a1dff4"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11023","number":11023,"commit":"b782ac5120b4afc594dc746ce73166b36e407140"}],"number":11275}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/11277)
* Commit: 005b20a66220e4655c7c1f6a7a308514cd0c8b45

### History
* :broken_heart: [Build #11275](https://buildkite.com/elastic/integrations/builds/11275) failed 48337698515e1cbf889045c7ee352a5ddceb7101
* :green_heart: [Build #11044](https://buildkite.com/elastic/integrations/builds/11044) succeeded 41510b0671eb1dc2fdc49c9cbd7e8e3be8a1dff4
* :green_heart: [Build #11023](https://buildkite.com/elastic/integrations/builds/11023) succeeded b782ac5120b4afc594dc746ce73166b36e407140

cc @milan-elastic

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11277","number":11277,"commit":"005b20a66220e4655c7c1f6a7a308514cd0c8b45"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11275","number":11275,"commit":"48337698515e1cbf889045c7ee352a5ddceb7101"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11044","number":11044,"commit":"41510b0671eb1dc2fdc49c9cbd7e8e3be8a1dff4"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11023","number":11023,"commit":"b782ac5120b4afc594dc746ce73166b36e407140"}],"number":11277}
buildkite-pr-comment-->

drewdaemon commented on packages/system/kibana/dashboard/system-Metrics-system-overview.json:
```diff
@@ -645,31 +645,32 @@
                                                     {
                                                         "color": "#cc5642",
-                                                        "stop": 100
+                                                        "stop": 1.85
```
Should this be `1.0`?

milan-elastic replied:
@drewdaemon  Yes, Ideally it should be `1` instead of `1.85`, seems like it has calculated it's value automatically! From the UI there is no place I've found from where I can manipulate this value! To make it `1` I've changed the value manually in json file.

drewdaemon replied:
@milan-elastic I'm sorry... I misunderstood. If this is the value the Kibana automatically set, we should stick to that. Let's revert to `1.85`. 

I think Kibana is probably doing its best to make sure that we still get a red color if the percentage value rises above `1`

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/11374)
* Commit: 5b98008ed7bb80bdcd1cc721f6055c5b264bcda8

### History
* :green_heart: [Build #11277](https://buildkite.com/elastic/integrations/builds/11277) succeeded 005b20a66220e4655c7c1f6a7a308514cd0c8b45
* :broken_heart: [Build #11275](https://buildkite.com/elastic/integrations/builds/11275) failed 48337698515e1cbf889045c7ee352a5ddceb7101
* :green_heart: [Build #11044](https://buildkite.com/elastic/integrations/builds/11044) succeeded 41510b0671eb1dc2fdc49c9cbd7e8e3be8a1dff4
* :green_heart: [Build #11023](https://buildkite.com/elastic/integrations/builds/11023) succeeded b782ac5120b4afc594dc746ce73166b36e407140

cc @milan-elastic

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11374","number":11374,"commit":"5b98008ed7bb80bdcd1cc721f6055c5b264bcda8"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11277","number":11277,"commit":"005b20a66220e4655c7c1f6a7a308514cd0c8b45"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11275","number":11275,"commit":"48337698515e1cbf889045c7ee352a5ddceb7101"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11044","number":11044,"commit":"41510b0671eb1dc2fdc49c9cbd7e8e3be8a1dff4"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11023","number":11023,"commit":"b782ac5120b4afc594dc746ce73166b36e407140"}],"number":11374}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/11377)
* Commit: af90798da6d5849003cf59f65336c9bd5028073b

### Failed CI Steps
* [Check integrations system](https://buildkite.com/elastic/integrations/builds/11377#018f714d-469a-41f2-af60-e1302ffdf90a)

### History
* :green_heart: [Build #11374](https://buildkite.com/elastic/integrations/builds/11374) succeeded 5b98008ed7bb80bdcd1cc721f6055c5b264bcda8
* :green_heart: [Build #11277](https://buildkite.com/elastic/integrations/builds/11277) succeeded 005b20a66220e4655c7c1f6a7a308514cd0c8b45
* :broken_heart: [Build #11275](https://buildkite.com/elastic/integrations/builds/11275) failed 48337698515e1cbf889045c7ee352a5ddceb7101
* :green_heart: [Build #11044](https://buildkite.com/elastic/integrations/builds/11044) succeeded 41510b0671eb1dc2fdc49c9cbd7e8e3be8a1dff4
* :green_heart: [Build #11023](https://buildkite.com/elastic/integrations/builds/11023) succeeded b782ac5120b4afc594dc746ce73166b36e407140

cc @milan-elastic

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11377","number":11377,"commit":"af90798da6d5849003cf59f65336c9bd5028073b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11374","number":11374,"commit":"5b98008ed7bb80bdcd1cc721f6055c5b264bcda8"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11277","number":11277,"commit":"005b20a66220e4655c7c1f6a7a308514cd0c8b45"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11275","number":11275,"commit":"48337698515e1cbf889045c7ee352a5ddceb7101"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11044","number":11044,"commit":"41510b0671eb1dc2fdc49c9cbd7e8e3be8a1dff4"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11023","number":11023,"commit":"b782ac5120b4afc594dc746ce73166b36e407140"}],"number":11377}
buildkite-pr-comment-->

drewdaemon reviewed (approved):
Great work.

I recommend respecting the final color stop value from Kibana ([ref](https://github.com/elastic/integrations/pull/9771#discussion_r1598978809)). 

Approving to unblock.

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/11405)
* Commit: c9e8f5660244692ccd52ccf5813e1fe0996f5409

### History
* :broken_heart: [Build #11377](https://buildkite.com/elastic/integrations/builds/11377) failed af90798da6d5849003cf59f65336c9bd5028073b
* :green_heart: [Build #11374](https://buildkite.com/elastic/integrations/builds/11374) succeeded 5b98008ed7bb80bdcd1cc721f6055c5b264bcda8
* :green_heart: [Build #11277](https://buildkite.com/elastic/integrations/builds/11277) succeeded 005b20a66220e4655c7c1f6a7a308514cd0c8b45
* :broken_heart: [Build #11275](https://buildkite.com/elastic/integrations/builds/11275) failed 48337698515e1cbf889045c7ee352a5ddceb7101
* :green_heart: [Build #11044](https://buildkite.com/elastic/integrations/builds/11044) succeeded 41510b0671eb1dc2fdc49c9cbd7e8e3be8a1dff4

cc @milan-elastic

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11405","number":11405,"commit":"c9e8f5660244692ccd52ccf5813e1fe0996f5409"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11377","number":11377,"commit":"af90798da6d5849003cf59f65336c9bd5028073b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11374","number":11374,"commit":"5b98008ed7bb80bdcd1cc721f6055c5b264bcda8"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11277","number":11277,"commit":"005b20a66220e4655c7c1f6a7a308514cd0c8b45"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11275","number":11275,"commit":"48337698515e1cbf889045c7ee352a5ddceb7101"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11044","number":11044,"commit":"41510b0671eb1dc2fdc49c9cbd7e8e3be8a1dff4"}],"number":11405}
buildkite-pr-comment-->

ishleenk17 commented:
@milan-elastic : I hope the dashboards json is autogenerated from Kibana and not manipulated manually. 
As in changes done on dashboards and then exported to json. 
As I see in some comment a change was done manually. 

Nit: There are 3 files showing in diff due to extra line addition. Please take care of that.

milan-elastic commented:
> @milan-elastic : I hope the dashboards json is autogenerated from Kibana and not manipulated manually. As in changes done on dashboards and then exported to json. As I see in some comment a change was done manually.
> 

@ishleenk17 After @drewdaemon comment I've reverted the `1` to `1.85` that I have changed manually previously. So it's been taken care of.

> Nit: There are 3 files showing in diff due to extra line addition. Please take care of that.

I think we should not revert them back, because those changes are result of `elastic-package check` command. so everytime anyone is going to run this command this diff will be appear!

ishleenk17 reviewed (approved):
Looks good!

milan-elastic commented:
I require code owner review to merge this PR, can someone from @elastic/sec-linux-platform and @elastic/sec-windows-platform review this PR and provide approval ?
cc: @lalit-satapathy @ishleenk17

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/11804)
* Commit: e284471fb47d763d3a01feec26cc0358613112dc

### History
* :green_heart: [Build #11405](https://buildkite.com/elastic/integrations/builds/11405) succeeded c9e8f5660244692ccd52ccf5813e1fe0996f5409
* :broken_heart: [Build #11377](https://buildkite.com/elastic/integrations/builds/11377) failed af90798da6d5849003cf59f65336c9bd5028073b
* :green_heart: [Build #11374](https://buildkite.com/elastic/integrations/builds/11374) succeeded 5b98008ed7bb80bdcd1cc721f6055c5b264bcda8
* :green_heart: [Build #11277](https://buildkite.com/elastic/integrations/builds/11277) succeeded 005b20a66220e4655c7c1f6a7a308514cd0c8b45
* :broken_heart: [Build #11275](https://buildkite.com/elastic/integrations/builds/11275) failed 48337698515e1cbf889045c7ee352a5ddceb7101
* :green_heart: [Build #11044](https://buildkite.com/elastic/integrations/builds/11044) succeeded 41510b0671eb1dc2fdc49c9cbd7e8e3be8a1dff4

cc @milan-elastic

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11804","number":11804,"commit":"e284471fb47d763d3a01feec26cc0358613112dc"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11405","number":11405,"commit":"c9e8f5660244692ccd52ccf5813e1fe0996f5409"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11377","number":11377,"commit":"af90798da6d5849003cf59f65336c9bd5028073b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11374","number":11374,"commit":"5b98008ed7bb80bdcd1cc721f6055c5b264bcda8"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11277","number":11277,"commit":"005b20a66220e4655c7c1f6a7a308514cd0c8b45"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/11275","number":11275,"commit":"48337698515e1cbf889045c7ee352a5ddceb7101"}],"number":11804}
buildkite-pr-comment-->

elasticmachine commented:
Package system - 1.58.1 containing this change is available at https://epr.elastic.co/search?package=system
